lorien/Assets/I18n/tr.txt

```diff
@@ -0,0 +1,139 @@
+LANGUAGE_NAME Turkish
```

Comment:
This is supposed to be written in the language itself...so the turkish word for the word "Turkish" :)

Reply:
Thank you for informing. I've fixed the issue.